tools/test

```diff
@@ -64,6 +64,7 @@ Suites:
   prettier
   deps
   tsflower
+  intl        (Doesn't run by default, as it takes several seconds.)
```

Comment:
Earlier in the usage message:

```
By default, run all suites, but only on files changed in this branch
as found by \`tools/git changed-files \$(tools/git base)\`.
```

So that should be updated too.